### ISSUE TITLE
 Invalid default value for 'modified_on'  [MYSQL 5.7.29]

### ISSUE THREAD

**Issue by M0ko** [directus/next#716]
Could not create a new projet with v9.0.0rc with MySql database
```
[ER_INVALID_DEFAULT] create table `directus_files` (`id` char(36) not null, `storage` varchar(255) not null, `filename_disk` varchar(255), `filename_download` varchar(255) not null, `title` varchar(255), `type` varchar(255), `folder` char(36), `uploaded_by` char(36
), `uploaded_on` timestamp not null default CURRENT_TIMESTAMP, `modified_by` char(36), `modified_on` timestamp, `charset` varchar(50), `filesize` int unsigned not null default '0', `width` int unsigned, `height` int unsigned, `duration` int unsigned, `embed` varcha
r(200), `description` text, `location` text, `tags` text, `metadata` json) - ER_INVALID_DEFAULT: Invalid default value for 'modified_on'

Please try again
```

**Comment by rijkvanzanten** [comment]
What version of MySQL are you using?

**Comment by M0ko** [comment]
MYSQL 5.7.29

**Comment by guilhermeaiolfi** [comment]
I hit that bug too.

10.4.14-MariaDB here.

**Comment by lluishi93** [comment]
Hi, IDK if this is going to help, but I just found about this:

![image](https://user-images.githubusercontent.com/14786440/96698658-b00eab80-138d-11eb-8aee-66022a543c6d.png)

I tried  using the following queries, and the project did install correctly:
![image](https://user-images.githubusercontent.com/14786440/96699763-f0225e00-138e-11eb-9570-23bf50544895.png)

The only "weird" thing is that the console  does  not exit once it shows this message:
![image](https://user-images.githubusercontent.com/14786440/96699962-2bbd2800-138f-11eb-81eb-65f5e8b59df5.png)

**Comment by benhaynes** [comment]
@lluishi93 interesting! That seems very familiar, and is something I've run into randomly with different MySQL databases. Let's see if Rijk's PR solves this, and if not we can either look into running these commands in Directus, or adding this to the docs/FAQs.

**Comment by guilhermeaiolfi** [comment]
There is something about not being able to have two timestamp fields with CURRENT_TIMESTAMP.  I've tried this fix and it didn't worked for me.

**Comment by rijkvanzanten** [comment]
@guilhermeaiolfi That's very strange, that must be a very specific MariaDB version limitation or something 🤔 i haven't heard of that problem before

MySQL 8 has seemingly smoothed out a lot of these inconsistencies and weirdnesses. I'm not unwilling to make MySQL 8+ the requirement for Directus either.. 👀

**Comment by rijkvanzanten** [comment]
The issue in the OP has been resolved in #746. @guilhermeaiolfi feel free to open a new issue detailing your exact setup and the exact issue you're getting if it still exists in the latest version 👍
